src/themes/OLH/templates/journal/search.html

```diff
@@ -10,12 +10,21 @@
 
 {% block body %}
 
+    
+
     <section id="content">
+
         <div class="row">
+            {%if search_term%}
+                <h5>Searching for: <i>{{search_term}}</i></h5>
+            {%elif keyword%}
+                <h5>Keyword: <i>{{keyword}}</i></h5>
+            {%endif%}
```

Comment:
inconsitent spacing in this block: Should be `{% foo %}` and `{{ bar }}`
(this applies to the changes on all three themes)